src/Tactic/Cong.agda

```diff
@@ -0,0 +1,185 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Macros for making working with cong more convenient
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Level renaming (suc to lsuc; _⊔_ to lmax)
+open import Relation.Binary.Definitions
+open import Tactic.Cong.Id
+open import Data.Nat renaming (_≟_ to _≟ℕ_)
```

Comment:
Imports should be in alphabetical order. Same comment below.

Reply:
Will do.